### DISCUSSION TITLE
CF3ENGINE ERROR

### DISCUSSION THREAD

**Discussion by chandresh1992** [cfengine/core#5401] [Q&A] [answered]
FACING SOME ISSUE WHILE STARTING SERVICE 
Starting cf-hub...                                         [  OK  ]
   error: 1050 licenses expired on 2016-08-31 -- reverting to Free25 Edition
mdb.c:5703: Assertion 'IS_LEAF(mp)' failed in mdb_cursor_next()
/bin/bash: line 1:  4734 Aborted                 /var/cfengine/bin/cf-hub
Starting cf-execd...                                       [FAILED]

if  anyone can help. Thanks in Advance

**Comment by olehermanse** [comment, marked as answer]
Hello,

It looks like you are using CFEngine Enterprise with an expired license. Enterprise customers get support via our support portal:

https://support.northern.tech/

If you don't have access to support, you can contact us via our website:

https://cfengine.com/contact/

**Reply by chandresh1992** [reply]
Hi, 
Thanks for your reply. we were using enterprises but the license expired a few years back. All service was working 
2 weeks before suddenly we are getting mail and checking on the policy server some services is not starting. 

when starting the service I got the below error. 
mdb.c:5703: Assertion 'IS_LEAF(mp)' failed in mdb_cursor_next()

**Reply by olehermanse** [reply]
@chandresh1992 Okay, it looks like you still have Enterprise installed. Please reach out to us via our website so we can help you sort this out:

https://cfengine.com/contact/

**Reply by chandresh1992** [reply]
Request done by https://cfengine.com/contact/

Can you please help us share some docs or any troubleshooting methods so that we can fix the issue? we are on production issue.

**Reply by olehermanse** [reply]
Hi.

We didn't receive your request. Please try again or reach us at contact (at) cfengine [dot] com, and we will have someone respond and help you.

**Reply by cody-valle** [reply]
Hi @chandresh1992  - I still haven't seen any Contact Us submissions from you come through yet. I'm happy to help you directly. You can email me at cody [dot] valle [at] northern [dot] tech.

**Comment by chandresh1992** [comment]
![Uploading image.png…]()

**Comment by chandresh1992** [comment]
![image](https://github.com/cfengine/core/assets/97735408/44e6b5ac-1d17-48e9-b039-e75d2f2004e6)

**Comment by chandresh1992** [comment]
We don't have any login details.

**Reply by olehermanse** [reply]
Alright, we responeded over email to a form submission now. If this was not you, feel free to email us at this address:

contact (at) cfengine [dot] com

**Reply by chandresh1992** [reply]
[support@northern.tech](mailto:support@northern.tech)
mail sent to the above mail ID.

**Reply by chandresh1992** [reply]
Hi,
Any update

**Reply by chandresh1992** [reply]
if possible please share us some docs so that we can fixed the issue.

**Reply by nickanderson** [reply]
All documentation can be found at https://docs.cfengine.com 

Support for CFEngine Enterprise requires a valid license and support contract and is handled via our support portal at https://support.northern.tech
